source/Crew.cpp

```diff
@@ -0,0 +1,242 @@
+/* Crew.cpp
+Copyright (c) 2019 by Luke Arndt
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "Crew.h"
+#include "GameData.h"
+
+using namespace std;
+
+// Load definition of a crew member
+void Crew::Load(const DataNode &node)
+{
+	// Set the id of this crew member so we know that we have loaded it.
+	if(node.Size() >= 2)
+		id = node.Token(1);
+	
+	for(const DataNode &child : node)
+	{
+		if(child.Size() >= 2)
+		{
+			// The crew members' display name (plural, Title Case)
```

Comment:
These comments should go above the variables' declarations (i.e. in the header file).